include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp

```diff
@@ -6,16 +6,13 @@
 #include "ck_tile/core.hpp"
 #include "ck_tile/ops/gemm/warp/warp_gemm_dispatcher.hpp"
 
+#define MEM_VEC_SIZE 16
```

Comment:
To avoid name collision, can we define this in the policy (as static data member)? The other macro `VectorLoadSize` should be moved as well.

Reply:
Thank you for the comments Poyen. Sure, I will rename the VectorLoadSize. I will create the commit after I solve the merge conflict and the modification of the universal_gemm_pipeline. Thank you!